content/en/tracing/other_telemetry/connect_logs_and_traces/go.md

```diff
@@ -47,12 +47,43 @@ func handler(w http.ResponseWriter, r *http.Request) {
 
 The above example illustrates how to use the span's context in the standard library's `log` package. Similar logic may be applied to third party packages too.
 
+## Injection into logrus logs
+
+An hook for the logrus package is available to automatically link your log and spans.
+The package is available in the Go tracer.
+
+```go
+package main
+
+import (
+    "github.com/sirupsen/logrus"
+
+    dd_logrus "gopkg.in/DataDog/dd-trace-go.v1/contrib/sirupsen/logrus"
+    "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+)
+
+func main() {
+    // Optional: Change log format to use JSON (Cf. Go Log Collection)
+    logrus.SetFormatter(&logrus.JSONFormatter{})
+
+    // Add Datadog context log hook
+    logrus.AddHook(&dd_logrus.DDContextLogHook{}) 
+    
+    // ...
+}
+```
+
+This automatically injects the trace id to your logs when you log with the context.
+```go
+    // Log with the context
+    logrus.WithContext(ctx).Info("Go logs and traces connected!")
+```
+
 **Note**: If you are not using a [Datadog Log Integration][1] to parse your logs, custom log parsing rules need to ensure that `dd.trace_id`, `dd.span_id`, `dd.service`, `dd.env` and `dd.version` are being parsed as strings. More information can be found in [Correlated Logs Not Showing Up in the Trace ID Panel][2].
```

Comment:
This line can be sent to the top of your changelist to make it more clear since this is the direct consequence of line 48

Reply:
Thanks! Applied in 1b0d827